check_code_issue.go

```diff
@@ -0,0 +1,28 @@
+package opslevel
+
+type CodeIssueCheckFragment struct {
+	Campaign string `graphql:"campaign"` // The campaign the check belongs to.
```

Comment:
This does not feel correct - why was this added?  (This feels like its a field on the base check type)

Reply:
Also where is `Constraint` and `IssueName` and `ResolutionTime` and `MaxAllowed` etc etc.

These are surely in the response payload.

Reply:
Oddly enough they are not. I'll ask around and see if these were forgotten in the API